src/components/ConnectionError/ConnectionError.tsx

```diff
@@ -1,11 +1,28 @@
 import { Button, Notification, Strip } from "@canonical/react-components";
 import type { PropsWithChildren } from "react";
+import reactHotToast from "react-hot-toast";
 
+import ToastCard from "components/ToastCard/ToastCard";
 import { getConnectionError } from "store/general/selectors";
+import { getAuditEventsErrors } from "store/juju/selectors";
 import { useAppSelector } from "store/store";
 
 const ConnectionError = ({ children }: PropsWithChildren) => {
   const error = useAppSelector(getConnectionError);
+  const auditLogsErrors = useAppSelector(getAuditEventsErrors);
+
+  if (auditLogsErrors) {
```

Comment:
I think this should be in a `useEffect` in case this component rerenders while there are audit log errors as this would cause a new toast to appear on every render (currently there shouldn't be many cases where it will rerender but this could change).

Reply:
Done!